runtime/doc/options.txt

```diff
@@ -2253,6 +2253,18 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option is reset when the 'paste' option is set and restored when
 	the 'paste' option is reset.
 
+					*'exrc'* *'ex'* *'noexrc'* *'noex'*
+'exrc' 'ex'		boolean (default off)
+			global
+	Enables the reading of .nvimrc and .exrc files in the current
+	directory.
+
+	The file is only sourced if the user indicates the file is trusted.
+	See |vim.secure.read()|.
```

Comment:
The hash check is worth mentioning here too.

Reply:
33c1e7ec33d3502bd779849a89a8411275438e1f